pulumi-azure-functions/deploy.mjs

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env zx
```

Comment:
@srmagura I never heard about it. But it looks pretty cool! Like something we’ve been missing for so much time.